### DISCUSSION TITLE
add automatically 'px'

### DISCUSSION THREAD

**Discussion by zhangenming** [vuejs/rfcs#524] [General]
### What problem does this feature solve?

```
  <div
    :style="{
      padding: '5px',
      top: top + 'px',
      left: left+ 'px',
    }"
  >
  </div>
```

### What does the proposed API look like?

```

  <div
    :style="{
      padding:  5,
      top: top,
      left,
    }"
  >
  </div>
```

**Comment by Shyam-Chen** [comment]
Use `vite-plugin-vue-inline-style`.

**Comment by Justineo** [comment]
F.Y.I. earlier discussions at https://github.com/vuejs/vue/issues/8759

**Comment by zhangenming** [comment]
imo, vue is vue because it provides a lot of simple things

**Comment by Justineo** [comment]
The reason for not implementing this: https://github.com/vuejs/vue/issues/8759#issuecomment-418598816

But given that React's adoption is so worldwide, it has already made it impossible for CSS to introduce changes that may break React. So I think it would be safe for other frameworks to introduce this feature.
